### ISSUE TITLE
Screensaver?

### ISSUE THREAD

**Issue by westurner** [sugarlabs/sugar#927]
Is there a place to configure a screensaver/screenlock?

xlock, i3lock, gnome-screensaver?

**Comment by quozl** [comment]
No, there isn't.  On OLPC OS there was no authentication or authorisation of user, so no password.  The olpc-powerd package was responsible for automatic suspend after a short time without user activity.  It kept the display active for some minutes, with the CPU suspended.  Then the CPU was briefly woken to turn off the display.  Sugar has not grown that feature set.

**Comment by westurner** [comment]
I've installed on Fedora 32 with `dnf install -y @sugar-desktop-environnent`; I assume this is similar to the Sugar on a Stick deployment platform?

(That's a neat feature to have that in BIOS. Whatever happened to Pixel Qi? I considered a Motion Ink Adam tablet with the transflective e-ink display. AFAIU, color e-ink is OTW, but the refresh rate is still slow.)

Is there a way to launch sugar in a window (in gnome) at least for development (instead of by selecting sugar as an alternate window manager/launcher in the GDM login screen)? That would solve for: screensaver with lock screen, sugar not working with e.g. LightDM on old notebooks, development and testing.

A person could require the window to launch in fullscreen in kiosk mode if that's necessary.

**Comment by westurner** [comment]
(There are now $100 [$200] Pinebook [Pro] ARM notebooks (and a tablet with a *detachable* backlit keyboard) for which there are now conda-forge packages like JupyterLab, IPyWidgets, and Matplotlib, that are installable with miniforge)

**Comment by srevinsaju** [comment]
> Is there a way to launch sugar in a window (in gnome) at least for development (instead of by selecting sugar as an alternate window manager/launcher in the GDM login screen)? That would solve for: screensaver with lock screen, sugar not working with e.g. LightDM on old notebooks, development and testing.

you might be interested in [sugar-runner](https://github.com/sugarlabs/sugar-runner.git) (unmaintained) and a [sugar docker image :whale: ](https://github.com/srevinsaju/sugar-runner)

**Comment by quozl** [comment]
I agree with @srevinsaju, it sounds like you may be interested in sugar-runner.  It's a developer hack though, and different to the experience designed for elementary school children.
